plugin/keyring_vault/vault_keys_container.h

```diff
@@ -0,0 +1,27 @@
+#ifndef MYSQL_VAULT_KEYS_CONTAINER_H
```

Comment:
In keyring framework the design is to use plural - for instance keys_container. If I change it to singular now it will break the naming convention from keyring framework.